### ISSUE TITLE
[Request] Set colour of pinned windows in task bar

### ISSUE THREAD

**Issue by horusofoz** [saif-ellafi/foundryvtt-window-controls#33]
Can we add an option to set a colour for pinned items?
![image](https://github.com/saif-ellafi/foundryvtt-window-controls/assets/3202339/1ba1022e-8d31-477f-a481-a98502369734)

**Comment by saif-ellafi** [comment]
Hi @horusofoz - If you are using Window Controls with Minimal UI, you can set the colors through Minimal UI color settings.

**Comment by saif-ellafi** [comment]
![image](https://github.com/saif-ellafi/foundryvtt-window-controls/assets/27952699/a561832d-35bc-495b-93d5-35e402b95215)

**Comment by horusofoz** [comment]
That works thank you. 

The Border Colors label for the color picker is a little confusing. Can I suggest changing it to something indicating it applies to pinned window title bars?

Thanks again :)

**Comment by saif-ellafi** [comment]
Yes it is a bit weird, because color configuration is from Minimal UI, which doesn't know anything about pinned windows and title bars. It is just a feature of "Window Controls" that it can get the color setting from Minimal UI :D 

Will give it some thought
